start/sw/web-app/nginx.md

```diff
@@ -0,0 +1,25 @@
+---
+title: NGINX HTTP Server
+subsection: web-app
+order: 7
+---
+
+# What is NGINX
+
+NGINX is a free, open-source, high-performance HTTP server and reverse proxy, as well as an IMAP/POP3 proxy server. NGINX is known for its high performance, stability, rich feature set, simple configuration, and low resource consumption. Website: [nginx.org](https://nginx.org/), [nginx.com](https://www.nginx.com/).
+
+## Install NGINX Web Server
+
+```bash
+sudo dnf install nginx
```

Comment:
Please use the `$` prepending each command([reference](http://developer-portal.github.io/contributing/write-introduction.html)).